scripts/bravo_csv.py

```diff
@@ -317,23 +318,23 @@ def default_bravo(lims, currentStep, with_total_vol=True):
                     if with_total_vol:
                         try:
                             # might not be filled in
-                            final_volume = art_tuple[1]['uri'].udf["Total Volume (uL)"]
+                            art_tuple[1]['uri'].udf["Total Volume (uL)"]
```

Comment:
Can we not have a `udf.get("Total Volume (uL)", None)` and check for `None` instead of a try-except?

Reply:
Fixed as suggested.